docs/3.0/develop/transactions.mdx

```diff
@@ -139,6 +308,9 @@ from time import sleep
 from prefect import task, flow
 from prefect.transactions import transaction
 
+@task
```

Comment:
Is this section supposed to be here? Looks like maybe not finished.

Reply:
Nope, didn't mean to update this section at all. I've removed my errant changes.